### ISSUE TITLE
Absolute Basic Install - something went wrong

### ISSUE THREAD

**Issue by andyseabrook007** [focustense/easymod#120]
So created an absolutely minimalist setup as I have still never been able to get this mod working. New instance in MO2 with only Consistant older people, USSEP, snd SKSE64 Data, Dawnguard, Hearthfire and Dragoborn DLCs and got the something went wrong message. Removed Consistant Older People and tried again, it still failed. Hope this helps.
[Log_20210924_072301_9944654.txt](https://github.com/focustense/easymod/files/7223380/Log_20210924_072301_9944654.txt)

**Comment by focustense** [comment]
Dupe of #103. Please refer to the [current resolution](https://github.com/focustense/easymod/issues/103#issuecomment-926757325) to get things running again.

The log posted here reports being v0.9.3 where the issue is fixed, but I am assuming you were running v0.9.2 or below at some point and tried to import a profile using that version. If you are positive that you never ran an older version, let me know, as it would be a serious problem if this issue is still occurring in new versions. My interpretation right now is that you updated to v0.9.3 somewhat recently but carried over the autosave corruption from an older version.

**Comment by andyseabrook007** [comment]
Well in fact I created a completely new Global instance in MO2 - so using a new default profile, intending to copy in all my NPC mods, specifically to remove other influences and be sure of firm ground in the use of EasyNPC. So created solely for your tool.
I have copied in the set of Mugshots but I think that is just the png's or am I mistaken. So would this not be 0.9.3 pure. There are no saves as I have never run the game in this instance/profile. - Andy
